pkg/controller/podautoscaler/OWNERS

```diff
@@ -2,14 +2,12 @@
 
 reviewers:
   - josephburnett
-  - MaciekPytel
-  - mwielgus
 approvers:
   - josephburnett
-  - MaciekPytel
-  - mwielgus
```

Comment:
While @josephburnett has been doing the majority of reviews/coding here I think it would be good if any of the autoscaling SIG-Leads had approval rights in this controller (please keep me and/or add @gjtempleton).

Reply:
Please open a new PR for adding folks @mwielgus. Looking forward to cleaned up roster so PR and issues can  be tagged correctly to fresh approvers/reviewers.